### PR TITLE
Make "fingerprint-reader" build

### PR DESCRIPTION
**Pull request by lslezak**

### CODE REVIEW

File: patches/fingerprint-reader.patch [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
++#include $(top_srcdir)/autodocs-ycp.ami
+diff --git a/src/clients/users_plugin_fingerprint_reader.ycp b/src/clients/users_plugin_fingerprint_reader.ycp
+index 1763905..dee084e 100644
+--- a/src/clients/users_plugin_fingerprint_reader.ycp
++++ b/src/clients/users_plugin_fingerprint_reader.ycp
+@@ -43,6 +43,7 @@
+     import "Users";
+     import "UsersPluginFingerprintReader";
+     import "Wizard";
++    import "Users";
```
Comment (jreidinger) [inline patches/fingerprint-reader.patch]: This looks like upstream bug?

Comment (lslezak) [inline patches/fingerprint-reader.patch]: Maybe, I don't know how exactly the user's plugins work. @jsuchome ??

Comment (jsuchome) [inline patches/fingerprint-reader.patch]: What's the bug, the double import? Where do you get it from, it is not in https://github.com/yast/yast-fingerprint-reader/blob/master/src/users_plugin_fingerprint_reader.ycp ?

Comment (jreidinger) [inline patches/fingerprint-reader.patch]: Problem is that I don't get why lslezak add additional users import.

Comment (jsuchome) [inline patches/fingerprint-reader.patch]: Well, that does not make sense to me neither

**Comment by lslezak** [comment]
I messed it up, I'll create a new PR...

**Pull request closed without merging**